package.json

```diff
@@ -12,6 +12,8 @@
     "dotenv": "^16.0.1",
     "express": "^4.18.1",
     "express-session": "^1.17.3",
+    "google-map-react": "^2.2.0",
+    "google-maps-react": "^2.0.6",
```

Comment:
these should be in the package.json in the frontend directory, not the root directory. but that causes a problem - it seems like `google-maps-react` doesn't support React 18, which is what we have. we don't want to downgrade React. can you look into this?

Reply:
this package supports React 18: https://www.npmjs.com/package/@react-google-maps/api

Reply:
ok, ill do this